index.html

```diff
@@ -0,0 +1,590 @@
+
+    <!-- Google Fonts -->
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500|Flamenco|Josefin+Sans:300,400|Nova+Mono|Poiret+One">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet"
+          href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css"
+          integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B"
+          crossorigin="anonymous">
+
+    <!-- FontAwesome CSS -->
+    <link rel="stylesheet"
+          href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+          integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+          crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/util.css">
+    <link rel="stylesheet" href="css/code.css">
+    <link rel="stylesheet" href="css/features.css">
+    <link rel="stylesheet" href="css/mockup.css">
+    <link rel="stylesheet" href="css/navbar.css">
+    <link rel="stylesheet" href="css/typography.css">
+
+    <script src="js/scroll.js"></script>
+  </head>
+
+  <body class="bg-light text-center">
+
+    <!-- Navbar -->
+
+    <nav class="navbar navbar-expand-md navbar-light bg-light">
+      <div class="container">
+        <div class="d-flex flex-row align-items-center navbar-brand mx-auto">
+          <img class="navbar-brand-img d-none d-md-block mr-2" src="img/logo.svg">
+          <span>polybar</span>
+        </div>
+        <div class="collapse navbar-collapse">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="https://github.com/jaagr/polybar">GitHub</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="https://github.com/jaagr/polybar/wiki">Docs</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="https://github.com/jaagr/polybar/issues/new/choose">Submit an Issue</a>
+            </li>
+          </ul>
+        </div>
+
+      </div>
+    </nav>
+
+    <!-- Hero -->
+
+    <div class="hero">
+      <div class="container">
+        <section class="row">
+          <div class="col-md-12">
+            <div class="content d-flex flex-column justify-content-between align-items-center">
+
+              <div>
+                <!-- This div is aligned to the top by justify-between -->
+              </div>
+
+              <header>
+                <h1 class="display-3 mb-sm-2 mb-lg-4">
+                  A fast and easy to use tool for creating<br class="d-none d-sm-block d-md-none"> status bars
+                </h1>
+                <h6 class="display-6 mb-sm-2 mb-lg-4">
+                  Polybar aims to help users build beautiful and highly customizable status bars for their desktop environment,
+                  <br class="d-none d-lg-block">
```

Comment:
Same question here.

Wouldn't it be cleaner to set the width to `80%` and `margin:auto` to center it and not have it be full length?